exonum/src/explorer/mod.rs

```diff
@@ -553,6 +553,18 @@ impl<'a> BlockchainExplorer<'a> {
         Some(TransactionInfo::Committed(tx))
     }
 
+    /// Return status of call in block
+    pub fn call_status(
```

Comment:
I'm against exposing this method since it's ambiguous for `before_transactions` / `after_transactions` hooks whether the call was executed successfully or not executed at all. It could be possible to use more low-level methods in the core schema instead.